scripts/publish.sh

```diff
@@ -7,4 +7,4 @@ set -e
 # # add `--dry-run` if you don't want it to do anything
 
 # Real Command
-poetry publish --build --username __token__ --password $PYPI_PASSWORD
+poetry publish --build --username __token__ --password $PYPI_TOKEN
```

Comment:
Is there any reason why the variable name is changed to token while the argument still has `--password`?

Reply:
haha what a nitpick :-D

Reply:
That's all on @htahir1 :)